nn.py

```diff
@@ -199,6 +222,7 @@ def __init__(self, device):
         self.fc3 = nn.Linear(512, 11)
         self.fc4 = nn.Linear(512, 7)
```

Comment:
Same as above: call these `fc1`, `fc2`, and `fc3`. (couldn't select the line with `self.fc2` in review mode)